modules/google-analytics.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Module Name: Google Analytics
+ * Module Description: MODULE DESCRIPTION COPY.
+ * First Introduced: 4.4
+ * Sort Order: 37
+ * Requires Connection: No
```

Comment:
I think it does require a connection. That feature won't be available in Dev mode.